<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainTheme</class>
 <widget class="QWidget" name="mainTheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb( 0, 0, 0);
</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 220);
 border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 3px;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::InputMethodHint::ImhLatinOnly|Qt::InputMethodHint::ImhMultiLine|Qt::InputMethodHint::ImhNoEditMenu</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:24pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>431</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1">
     <widget class="QPushButton" name="button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="multiplyButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="plusButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="minusButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="divButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="rigthBracketButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="leftBracketButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="deleteLastButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="document-revert"/>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="clearAllButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="buttonPoint">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 255, 255, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 215,215, 215, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="equalButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba( 255, 128, 0, 200);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: biege;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgba( 193, 94, 0, 200);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
